android/src/main/java/com/genesys/messenger/GenesysCloudChatActivity.java

```diff
@@ -0,0 +1,202 @@
+package com.genesys.messenger;
+
+import android.content.Intent;
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.integration.core.StateEvent;
+import com.nanorep.convesationui.messenger.MessengerAccount;
+import com.nanorep.convesationui.structure.controller.ChatController;
+import com.nanorep.convesationui.structure.controller.ChatEventListener;
+import com.nanorep.nanoengine.AccountInfo;
+import com.nanorep.sdkcore.utils.NRError;
+
+public class GenesysCloudChatActivity extends AppCompatActivity implements ChatEventListener {
+    private static final String TAG = GenesysCloudChatActivity.class.getSimpleName();
+    private static final String CONVERSATION_FRAGMENT_TAG = "conversation_fragment";
+
+    public static final String EXTRA_DEPLOYMENT_ID = "deploymentId";
+    public static final String EXTRA_DOMAIN = "domain";
+    public static final String EXTRA_TOKEN_STORE_KEY = "tokenStoreKey";
+    public static final String EXTRA_LOGGING = "logging";
+
+    private ChatController boldController;
```

Comment:
At the risk of making a trifling comment, it seems it might make sense for this var to be called `chatController` instead of `boldController`?

Reply:
yes we can

Reply:
Sure.

Reply:
🎉